py/desisim/scripts/quickquasars.py

```diff
@@ -614,6 +613,9 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
     else:
         log.info('Z_noRSD field not present in transmission file. Z_noRSD not saved to truth file')
 
+    if args.shift_kms_los:
+       metadata['Z']+=(args.shift_kms_los/c*(1.+metadata['Z']))
```

Comment:
`(` and `)` are superfluous.
`metadata['Z'] += args.shift_kms_los/c*(1.+metadata['Z'])`
is perfect